packages/theme/css/theme.css

```diff
@@ -35,3 +35,8 @@
 @import url(characteristics/status.css);
 @import url(characteristics/taggable.css);
 @import url(characteristics/text.css);
+
```

Comment:
Just curious, why was this put here instead of in global?

Reply:
Global is included when you import the theme anyway

Reply:
There was a discussion about having it as part of theme, but I think the point was more like having (maybe) some global styling per theme. Not sure if that's something we want to do. But either way it should be done in a different PR.